packages/firebase_remote_config/firebase_remote_config_platform_interface/lib/src/platform_interface/platform_interface_firebase_remote_config.dart

```diff
@@ -25,8 +23,8 @@ abstract class FirebaseRemoteConfigPlatform extends PlatformInterface {
 
   /// Create instance using [app] using the existing implementation.
   factory FirebaseRemoteConfigPlatform.instanceFor({
-    FirebaseApp app,
-    Map<dynamic, dynamic> pluginConstants,
+    required FirebaseApp app,
```

Comment:
This and `delegateFor`'s `app` should probably be optional, to avoid making a too big of a breaking change.